data.sql

```diff
@@ -10,4 +10,30 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 VALUES ('Pikachu', '2021-01-7', 15.04, 'false', 1);
 
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) 
-VALUES ('Devimon', '2017-05-12', 11, 'true', 5); 
\ No newline at end of file
+VALUES ('Devimon', '2017-05-12', 11, 'true', 5); 
+
+/*Day 2 of Populate database with sample data. */
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) 
+VALUES ('Charmander', '2020-02-08', -11, 'false', 0); 
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) 
+VALUES ('Plantmon', '2021-11-15', -5.7, 'true', 2); 
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) 
+VALUES ('Squirtle', '1993-04-02', -12, 'false', 3); 
```

Comment:
- [ ] @shegy28 kindly ensure your figures are correct , your weight is suppose to be  -12.13kg

Reply:
okay i will update it now